File: Sprint0/Java - Descartado/PruebasUnitarias/src/main/java/com/prueba/test/Distancia.java
package com.prueba.test;
import java.lang.Math;

public class Distancia {
    // Función que calcula la distancia de un punto al Origen
    public double calcularDistancia(Coordenada punto){
        return Math.sqrt(Math.pow(punto.x,2)+Math.pow(punto.y,2));
    }
}

File: Sprint0/PruebasUnitarias/SOSgame/Distancia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sprint0
{
    public class Distancia    {
        // Función que calcula la distancia de un punto al Origen
        public double CalcularDistancia(Coordenada punto)
        {
            return Math.Sqrt(Math.Pow(punto.getX(), 2) + Math.Pow(punto.getY(), 2));

        }
    }
}






